package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "monitor-uk",
+  "version": "1.0.0",
+  "description": "APC aggregation and reporting interface.",
+  "main": "Gruntfile.js",
+  "directories": {
+    "doc": "docs"
```

Comment:
It was a auto-generated file. For now we are using it only for grunt so I removed the directories from that file.